pkg/runtime/framework/plugins/plainml/plainml.go

```diff
@@ -50,13 +50,15 @@ func (p *PlainML) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob
 
 	// TrainJob contains the actual information for the number of nodes.
 	if trainJob.Spec.Trainer != nil && trainJob.Spec.Trainer.NumNodes != nil {
-		info.FindPodSetByName(constants.JobTrainerNode).Count = trainJob.Spec.Trainer.NumNodes
+		if trainerPS := info.FindPodSetByAncestor(constants.AncestorTrainer); trainerPS != nil && trainerPS.Count != nil {
```

Comment:
Do we need to check trainerPS.Count here ?

Reply:
Avoiding null pointer.